packages/pass-otp/build.sh

```diff
@@ -1,19 +1,21 @@
 TERMUX_PKG_HOMEPAGE=https://github.com/tadfisher/pass-otp
-TERMUX_PKG_DESCRIPTION="A pass extension for managing one-time-password (OTP) tokens"
+TERMUX_PKG_DESCRIPTION="A pass/passage extension for managing one-time-password (OTP) tokens"
 TERMUX_PKG_LICENSE="GPL-3.0-or-later"
+_COMMIT=a364d2a71ad24158a009c266102ce0d91149de67
+_COMMIT_DATE=20221228
 TERMUX_PKG_MAINTAINER="Henrik Grimler @Grimler91"
-TERMUX_PKG_VERSION=1.2.0
-TERMUX_PKG_REVISION=4
-TERMUX_PKG_SRCURL=https://github.com/tadfisher/pass-otp/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=5720a649267a240a4f7ba5a6445193481070049c1d08ba38b00d20fc551c3a67
-TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="oathtool, pass"
+TERMUX_PKG_VERSION=1.2.0-p${_COMMIT_DATE}
+TERMUX_PKG_SRCURL=https://github.com/tadfisher/pass-otp/archive/$_COMMIT.tar.gz
+TERMUX_PKG_SHA256=02c6fa59b30ee009c62807b763c41df44cc9e4da3e137f1b5774e995ee18e2f6
+TERMUX_PKG_DEPENDS="oathtool"
 TERMUX_PKG_SUGGESTS="libqrencode"
```

Comment:
The SUGGESTS field should probably be updated to suggest `pass` or `passage`.
```suggestion
TERMUX_PKG_SUGGESTS="pass | passage, libqrencode"
```

Reply:
Thanks, the SUGGESTS field shows that now.